src/main.cpp

```diff
@@ -451,7 +460,8 @@ void loop()
     send_all_CAN_msgs(CAN3_txBuffer, &TELEM_CAN);
 
     // Basic debug prints
-    if (curr_tick.triggers.trigger5)
+    // if (curr_tick.triggers.trigger5)
+    if (false)
```

Comment:
Ideally this is changed to be a build flag or flag defined somewhere central, probably fine for now